src/common/models/Projection.ts

```diff
@@ -510,9 +521,10 @@ export class Projection {
       metricsTimeseriesRaw,
     );
 
-    let currDate = earliestDate.clone();
-    while (currDate.diff(latestDate) <= 0) {
-      const ts = currDate.format('YYYY-MM-DD');
+    let currDate = new Date(earliestDate.getTime());
+
+    while (currDate <= latestDate) {
+      const ts = currDate.toISOString().substring(0, 10);
```

Comment:
The most strightforward way I could find to get the date in `YYYY-MM-DD` format in native JavaScript is to first convert it using `toISOString()` then take the first 10 characters 😐 . Probably not the most elegant but other methods seem unnecessarily lengthy.